assignment-2/7s.md

```diff
@@ -1,2 +1,293 @@
-#7S
-// TODO
\ No newline at end of file
+# 7S Framework #
+
+> Assignment - 7S:
+> Organisations are not just about methodology. There are quite a number of abstractions for organisations. We will look at the 7S model. Now we go and look outside to two successful companies through the eyes of the 7S model. You can select these companies. One should be a software company, one should come from another industry. What can you learn from this for your failed case? What can you learn from this about the methods RUP and Scrum.
+> 
+> Look for organisations that really make clear how they work from the inside. Think about Etsy, Quora, Google, Facebook, Apple, Microsoft, Mozilla, Amazon, Spotify.
+
+## 1. Introduction ##
+
+7S is a management model developed by  R.H. Waterman Jr. and T.Peters. The 7 Ss are structure, strategy, systems, skills, style, staff and shared values. The model is most often used as an organizational analysis tool to assess and monitor changes in the internal situation of an organization
+
+#### Principles
+(think about measures that will improve performance).
+The core idea lays on the question "How I distribute my sources in order to get the best results and create some value?".
+The seven elements need to be aligned and helping each other in order to hold performance at the host organization, and organizations must be align with external enviroment. 
+The model can help to identify non aligned issues at critical moments like during changes at the organization. Also to clarify some of the key issues that must be addressed for such organizations to be effective and adaptive.
+
+#### Process Area Descriptions
+
+##### Strategy :
+What makes the organization special?. Ones such question is answer of course based on the organization vison statement. The next step is to identify the kind of problems the organization addresses on its functional areas, aiming to enhance its competitive advantage. 
+
+##### Structure :
+How the organization is arranged or configured? For example some organizations are divided for their geographical distribution, others for its functionalities, others focussed on their main production, aiming to clarify its integration and coordination mechanisms.
+
+##### Systems :
+It refers to the formal procedures for measurement, rules and processes for getting things done and managing the activity of the organization. They include personnel evaluation , rewards, performance assessments, chain-ofcommand relationships, policies for planning, monitoring, and executing projects.
```

Comment:
"personnel evaluation  ," = "personnel evaluation," => No space before the comma